dashboard/test/ui/features/acquisition_products/pd/teacher_application.feature

```diff
@@ -100,7 +145,33 @@ Scenario: Teacher saves, re-opens, and submits an application
   And I see no difference for "Viewing previously-saved teacher application"
 
   # Finish application to submit it
-  And I complete Section 1 of the teacher PD application
+  And I press keys "Snape" for element "input#lastName"
```

Comment:
I have to say- it was very satisfying to see "Severus" and "Snape" appear during that Saucelabs run😂. That said- approved. I've had to add a no_mobile exclusion on ui tests before- do we have any idea how many teachers submit applications on mobile?

Reply:
Yes! And the principal is Dumbledore :) I may have to do that as well, but I'm going to look into it first just in case, and I don't know how many sorry :/

Reply:
Thank you for the quick review!